src/sass/feature/_base.scss

```diff
@@ -4,50 +4,52 @@
 @use 'core' as *;
 
 .#{$prefix}-feature {
+  display: flex;
+  flex-direction: column;
+
+  &--horizontal {
+    flex-direction: row;
+
+    .#{$prefix}-feature__image {
+      margin-right: $spacing-sm;
+    }
+  }
+
   &__image img {
     display: block;
     width: 100%;
   }
 
   &__body {
-    margin-top: $spacing-md;
+    * + * {
```

Comment:
Something to think about here is that using the universal version of this selector instead of the direct child version Eg. `> * + *` means that _anything_ inside the `_body` element will have the top margin applied no matter how deeply nested. I think in this case we only want consistent top margin applied to the direct children (`__detail1` elements) that we know about. Using the selector as is could have unintended consequences when someone puts multiple elements inside of a `__detail` element. I don't think we should be controlling the spacing between elements that people are putting in the "slots" we are providing.

here's an example. There is top margin being applied to the _Action_ you have inside this configuration causing the display of the component to be thrown off:

![image](https://user-images.githubusercontent.com/5590034/75355195-1b928e80-587c-11ea-8326-07489995a9ab.png)